components/MatchBanner.js

```diff
@@ -58,15 +68,21 @@ export default class MatchBanner extends React.Component {
 						: <Text>No one has submitted a bid yet, but be patient</Text>
 						}
 
-						<View>
+						<View> {/** Can we style this view, so that these buttons are in a row */}
```

Comment:
it should be doable with flexDirection: 'row' and flexWrap: 'nowrap' but cant say for sure cause styling is a bitch